src/account/AccountAllowanceAdjustTransaction.js

```diff
@@ -1,534 +1,534 @@
-import Transaction, {
```

Comment:
Unfortunately, because this is already on `main` and has been released we still need to keep this type around. What we should do instead of commenting everything out, is remove all our internal methods. This would make it so executing this request would error, but users can still construct and update this request otherwise.